spinflip/diagonalizer.py

```diff
@@ -89,3 +96,62 @@ def state_evolution_plotter(self, t_lim = 'timescale', resolution=500, quantity
         if savefig == True: 
             plt.tight_layout()
             plt.savefig('evolvedstate.png', dpi=300)
+        
+        f.show()
```

Comment:
I think it would be best to not have show() in the library itself, since we can't expect the function to always be called in an interactive setting. If you like, you can return f so the user can show it.

Reply:
true